test/system_test.py

```diff
@@ -355,6 +355,13 @@ def test_install_latest(self):
         spack_install_and_test('flexpart-ifs @meteoswiss-10')
 
 
+@pytest.mark.no_tsa  # flexpart-cosmo does not run on Tsa
```

Comment:
Why not?

Reply:
According to @pirmink, on tsa we would still have to stick with `eccodes` 2.19.0, but he doesn't use spack on tsa anyway (and won't start using it again in the rest of tsa's lifetime). So since nobody uses spack for flexpart on tsa, we don't need to test there.